e2e-tests/pipeline_validation.sh

```diff
@@ -135,21 +151,37 @@ function fhir_source_query() {
       enc_obs_query_param="?subject.given=Alberta625"
   fi
   
-  curl -L -X GET -u admin:Admin123 --connect-timeout 5 --max-time 20 \
-    "${FHIR_SERVER_URL}/Patient${patient_query_param}" 2>/dev/null >>"${HOME_PATH}/${PARQUET_SUBDIR}/patients.json"
+  if [[ -n ${JDBC_MODE_HAPI} ]]; then
+      patient_query_param="?_summary=count"
+      enc_obs_query_param="?_summary=count"
```

Comment:
We should probably do this for any FHIR server; does OpenMRS FHIR module not support `_summary=count`?